book/embeds.md

```diff
@@ -1290,10 +1273,10 @@ class Tab:
 ```
 
 When an iframe is clicked, it passes the click through to the child
-frame, and immediately return afterwards, because iframes capture
-click events. Note how I subtracted the absolute x and y offset of the
-iframe from the (absolute) x and y click position when recursing into the child
-frame:
+frame, and immediately returns afterward, because iframes capture
+click events. Note how I subtracted the absolute `x` and `y` offsets of the
```

Comment:
I think in my chapters I saw the copyeditor want "x" and "y" to be italics, as in `*x*`, not code font.

Reply:
Done